pseudoalgo.txt

```diff
@@ -0,0 +1,66 @@
+def generateMeals:
```

Comment:
save this file as a .py so that syntax highlighting can be shown.

Reply:
it's just pseudocode, and you can select the highlighting in the options at the bottom